profiles/Makefile.am

```diff
@@ -7,6 +7,7 @@ dist_profile_sssd_DATA = \
     $(top_srcdir)/profiles/sssd/password-auth \
     $(top_srcdir)/profiles/sssd/postlogin \
     $(top_srcdir)/profiles/sssd/README \
+    $(top_srcdir)/profiles/sssd/REQUIREMENTS \
```

Comment:
You also need to own the requirements in the specfile otherwise you get:
```
error: Installed (but unpackaged) file(s) found:
   /usr/share/authselect/default/sssd/REQUIREMENTS
   /usr/share/authselect/default/winbind/REQUIREMENTS
    Installed (but unpackaged) file(s) found:
   /usr/share/authselect/default/sssd/REQUIREMENTS
   /usr/share/authselect/default/winbind/REQUIREMENTS
```